docs/_includes/footer.html

```diff
@@ -1,24 +1,9 @@
-<footer class="site-footer">
-    <div class="wrapper">
-        <div class="footer-col-wrapper">
-            <div class="footer-col footer-col-1">
-                <ul class="contact-list">
-                    <li>{{ site.title }}</li>
-                    <li><a href="http://groups.google.com/group/sinonjs">http://groups.google.com/group/sinonjs</a></li>
```

Comment:
Are we sure we want to get rid of this? I think we should have at least the same pointers as we already have in the README on Github. If not in the same place, then somewhere.